meilisearch/index.py

```diff
@@ -1,24 +1,18 @@
+import urllib
 from meilisearch._httprequests import HttpRequests
-from meilisearch.update import Update
-from meilisearch.document import Document
-from meilisearch.search import Search
-from meilisearch.stat import Stat
-from meilisearch.setting import Setting
-
-# pylint: disable=too-many-ancestors
-class Index(Update, Document, Search, Stat, Setting):
+
+# pylint: disable=R0904
```

Comment:
what is this rule for?

Reply:
It disables a checking rule for too many public methods, because the linter limits them to 25 for a single class

Reply:
What would be the good way to separate the methods in multiple classes?

Reply:
I think we could totally refactor it in separate classes with some hierarchy, but the problem is that I didn't want to introduce any braking changes in this PR. When there is a new breaking version of MeiliSearch, we can restructure it completely. For now I think the most important thing was to get rid of the unnecessary complexity of the heritage, to make it accessible and readable, and for me it is not important to have less than 25 public methods in a class, if it makes the structure of the SDK simple and easy to use. Every way of putting a method or functionality in a different place (that I can think of) would introduce braking changes or a complex inheritance that doesn't add anything to the SDK. What do you think?